libs-scala/resources/src/main/scala/com/digitalasset/resources/ProgramResource.scala

```diff
@@ -59,5 +64,23 @@ class ProgramResource[T](
 }
 
 object ProgramResource {
-  abstract class SuppressedException extends RuntimeException
+
+  abstract class StartupException(message: String, cause: Throwable)
```

Comment:
```suggestion
  abstract class StartupException(override val getMessage: String) extends RuntimeException with NotStackTrace
```
Doesn't look like you are using it in any other way right now.

Reply:
If you choose to accept this suggestion you can drop the body of the class.

Reply:
Done. I thought I was being future-proof but you're right, it's not used.